examples/full-example/.mendelrc

```diff
@@ -210,4 +209,6 @@ env:
         outfile: server.js
         entries:
           - ./main.js
-        runtime: main # This
+        # If npm package has multiple entry files, this pulls only
+        # server side files defined in "main" field of package.json
+        runtime: main
```

Comment:
This is pretty hard to understand. I don't fully follow to be honest, although I do understand the concept of multiple runtimes package.json very well.

Maybe say something "`runtime` forces a particular bundle to a certain runtime, per CommonJS spec. See more at:" and find an useful link or write a new documentation on why this exists in Mendel.

The core suggestion here is: It is too much to explain on the full example, use something simple (maybe even simpler than my suggestion) and link to a full spec or full docs.